dev-docs/source/VSCode.rst

```diff
@@ -384,18 +380,16 @@ Install `ptvsd` using pip within the terminal
 
 .. code-block:: python
 
-    import ptvsd
-    ptvsd.enable_attach(address=('127.0.0.1', 5678), redirect_output=True)
-    ptvsd.wait_for_attach()
-    ptvsd.break_into_debugger()
+    import debugpy
+    debugpy.listen(('127.0.0.1', 5678))
+    debugpy.wait_for_client()
+    debugpy.breakpoint()
 
-- When Mantid appears to freeze. Open the debug tab and start the Python 
-  Attach Target
-- Any additional breakpoints using the IDE are added automatically 
+- When Mantid appears to freeze. Open the debug tab and start the "Python Attach" Target
+- Any additional breakpoints using the IDE are added automatically
   (i.e. don't add `ptvsd.break_into_debugger()`
```

Comment:
Shouldn't this line be changed to `debugpy.breakpoint()` ?

Reply:
Nice catch! Thanks. 
My editor search must have been broken that day :facepalm: